rfc/text/basic/bp_sessions.md

```diff
@@ -126,6 +126,10 @@ where
 
 No response to an `ABORT` message is expected.
 
+There are few scenarios, when `ABORT` is used:
+
+* During session establishment, if peer decided to abort connect. 
```

Comment:
Although not that important ... I'd prefer the parlance "session **opening**" over "session *establishment*" because we're talking about WAMP sessions which can be **opened** using HELLO/WELCOME exchanges (and thereafter *closed* using GOODBYE). 

Instead, WAMP connections are those which get *established* (and later on *dropped*/closed) by just using the underlying transport protocol (e.g. TCP or WebSocket).

Reply:
Agree.